lib/quby/questionnaires/entities/charting/chart.rb

```diff
@@ -10,9 +10,20 @@ class Chart
           attribute :key,                        Symbol
           attribute :title,                      String
           attribute :plottables,                 Array
-          # if categories are defined, plottable values should correspond to values from this array
-          # and the graph will  be plotted with matching y axis categories
+          # if y_categories are defined, plottable values should correspond to values from this array
+          # and the graph will be plotted with corresponding y axis categories
+          # example (icl_r): ["Zeer laag", "Laag", "Gemiddeld", "Hoog", "Zeer Hoog"] (caution, capitalization oddity)
+          # NB: only implemented for bar charts on the roqua side
           attribute :y_categories,               Array
+          # if y_range_categories are defined, plottable values should fall in the ranges that compose the keys of this
+          # hash. The chart will label these ranges of y_values with the corresponding value in the hash on the y axis.
+          # example:
+          # {(0..30) => "Zeer laag", (30..40) => "Laag", (40..60) => "Gemiddeld",
+          # (60..70) => "Hoog", (70..100) => "Zeer hoog"}
```

Comment:
Voelt als primitive obsession. 
Zou liever een Categories object hebben dat meteen kan checken of alles aansluitend is, of mogelijk zelfs geinitialiseert kan als `0, 'foo', 4, 'bar', 7, 'bla', 10`. Want zoals het nu gebruikt wordt zouden niet aansluitende ranges niet echt duidelijk worden in roqua. Dat object kan dan methodes hebben om per range een object terug te geven met attrs from, till, name. en een category_for, zodat we het ook makkelijk in scores kunnen gebruiken. 
En zou een as_json kunnen definiëren (later)

Is meer werk, maar voelt zo wel erg houtje touwtje en moeilijk aanpasbaar in de toekomst.